cdap-watchdog/src/main/java/co/cask/cdap/logging/LoggingConfiguration.java

```diff
@@ -51,7 +51,7 @@
   // Table used to store log metadata
   public static final String LOG_META_DATA_TABLE = "log.meta";
   // Defaults
-  public static final String DEFAULT_LOG_PATTERN = "%d{ISO8601} - %-5p [%t:%C{1}@%L] - %m%n";
+  public static final String DEFAULT_LOG_PATTERN = "%d{ISO8601} - %-5p [%t:%c{40}@%L] - %m%n";
```

Comment:
with the pattern ``[%t:%C{1}@%L]`` we will always print the className as the context, so when we emit the log from javascript, the context would be ``sun.reflect.NativeMethodAccessorImpl``, instead we want to have the transform context, that's why we have to change the pattern to be 
[%t:%c{40}@%L] , the 40 is to show the stage number as well as the class name {Eg: validatorTransform}